aws/rust-runtime/aws-config/src/ecs.rs

```diff
@@ -61,11 +61,15 @@ use tower::{Service, ServiceExt};
 
 use crate::http_credential_provider::HttpCredentialProvider;
 use crate::provider_config::ProviderConfig;
+use aws_smithy_client::http_connector::ConnectorSettings;
 use aws_types::os_shim_internal::Env;
 use http::header::InvalidHeaderValue;
 use std::time::Duration;
 use tokio::sync::OnceCell;
 
+const DEFAULT_READ_TIMEOUT: Duration = Duration::from_secs(5);
+const DEFAULT_CONNECT_TIMEOUT: Duration = Duration::from_secs(2);
```

Comment:
It'd be nice if all these defaults were documented in one place in our FAQ or something. Or perhaps they could be imported from a crate of constants?

Reply:
On second thought, I think defining these settings in the module where they are immediately relevant is the right choice, disregard my above comment.